### PR TITLE
[1328] Display course tabs in course details

### PR DESCRIPTION
**Pull request by tvararu**
### Context

We need to present the basic details on the frontend, and the enrichment information on the ui.

### Changes proposed in this pull request

- Remove the course details section
- Remove the training locations section
- Wrap remaining content in govuk-tabs CSS, but without the JS enabled (it's not needed)

### Guidance to review

Review commit by commit, ignoring whitespace.

### After

Screenshot 2019-04-17 at 18 12 24

### CODE REVIEW

**Comment by tvararu** [comment]
[Course details tabs (manage-ui)](https://trello.com/c/QV1UHxv7/1328-course-details-tabs-manage-ui)

**Comment by tvararu** [comment]
Note well: the https://github.com/DFE-Digital/manage-courses-ui/pull/328/commits/8d83de5e2ef5670ef006d1fd7bb34001ce1e77f7 commit removes the guard clauses around the enrichment information so that it's always displayed, so users don't get an empty page.

cc @benilovj @fofr

**Comment by tvararu** [comment]
Added a "do not merge" as this is the last step we should do in the courses work, as it links the ui to the frontend, which means the frontend needs to be fully ready for this to be shippable.

**Comment by fofr** [comment]
@tvararu Do you have a screenshot of how a not running course looks now?

**Comment by tvararu** [comment]
@fofr 

Screen Shot 2019-04-18 at 09 22 44 (2)

**Comment by tvararu** [comment]
Updated:

Screen Shot 2019-04-18 at 09 56 55 (2)

File: src/ui/Assets/Styles/site.scss [hunk 1]
```diff
@@ -106,3 +107,7 @@ $success-green: #00823b;
   @include govuk-font($size: 16, $line-height: 1.5);
   color: $govuk-text-colour;
 }
+
+.pointer-events-none {
```
Comment (fofr) [inline src/ui/Assets/Styles/site.scss]: This relates to this bug:
https://github.com/alphagov/govuk-frontend/issues/1252

**Pull request merged**